src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -5563,6 +5561,60 @@ def update_auto_upgrade_profile(self, mc: ManagedCluster) -> ManagedCluster:
             mc.auto_upgrade_profile.upgrade_channel = auto_upgrade_channel
         return mc
 
+    def update_identity(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
The implementation of this function basically follows [this part](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L2656-L2694) in file `custom.py`.